megapowerani.com.template.json

```diff
@@ -0,0 +1,70 @@
+{
+    "providerId": "megapowerani.com",
+    "providerName": "Freshsales Test",
+    "serviceId": "template",
+    "serviceName": "Domain verification",
+    "version": 1,
+    "logoUrl": "https://avatars.slack-edge.com/2021-08-02/2349087959585_442632783598439baea4_512.png",
+    "description": "Freshsales Domain Verification Test",
+    "variableDescription": "This is  atest for Freshsales domain verification using Domain Connect",
+    "syncPubKeyDomain": "megapowerani.com",
+    "syncRedirectDomain": "megapowerani.com",
+    "warnPhishing": true,
```

Comment:
Do I oversee something or there is no phishing vector from within this template?
There are no variables and signing in place so this template should be secure AFAIK.

Reply:
Yes, this is a static template only for a test.